src/clr-angular/forms/styles/_select.clarity.scss

```diff
@@ -84,6 +84,7 @@
     border: $clr-default-borderwidth solid $clr-forms-select-multiple-border-color;
     border-radius: $clr-global-borderradius;
     height: auto;
+    min-width: 5rem;
```

Comment:
I'm guessing here. It seems like before removing the deprecated css they expanded automatically. After removing the deprecated css they need the min width to look the same. Is that accurate?

Reply:
Correct, this line was in the depreacated css but applying everywhere. Without it all selects would be the browser default which is smaller than the 5rem minimum